### ISSUE TITLE
gui: Deal with translation sabotager

### ISSUE THREAD

**Issue by andronoob** [bitcoin/bitcoin#25821]
I'm sorry if this doesn't worth opening a new issue thread as it's already mentioned in #24987.

Bitcoin Core currently makes use of Transifex for localization/translation. However I can't see activity stream on Bitcoin Core's Transifex project, therefore it's difficult to track what recent changes of translated text have been made.

Recently zh_CN translation seems to be sabotaged, in the way that all the translated text strings are replaced with its corresponding original English text.

To make translation contributors' life easier, I think something should be done, like what Electrum's project hosted on Crowdin currently enjoys: the activity stream feature which allows effective tracking of translated text.

**Comment by fanquake** [comment]
cc @hebasto

**Comment by hebasto** [comment]
It is unfortunate that Transifex.com have no means to raise an issue, such as this on, and track it. Therefore, I think it is ok to bring it here, or in the [GUI](https://github.com/bitcoin-core/gui) repo.

We experienced similar issues at lease two times in the past, unfortunately. French and Ukrainian translations were the targets. It is still unclear what was that -- a sabotage, an act of vandalism or a blunder.

In both previous cases translation teams managed to fix everything themselves. French team also contacted to Transifex support.

Actually, it is translation team's responsibility. The [`zh_CN` team](https://www.transifex.com/bitcoin/teams/159/zh_CN/) has 197 translators, and [seone](https://www.transifex.com/user/profile/seone/) as a coordinator who has not been active for years. Why not nominate a new person or a few people for coordinator role on the IRC chat?

FWIW, after the issue with Ukrainian translation, I've volunteered for a coordinator role in the Ukrainian team. And now I have backups of translations.

> Besides, I hope the sabotage done to zh_CN translation could be reverted in some easy way.

I guess it could be possible to restore translations from the `src/qt/locale/bitcoin_zh_CN.ts`. But it definitely requires the collaboration of the `zh_CN` team translators.
